### ISSUE TITLE
Cannot import data due to non-existent unique constraint in 1.3.3

### ISSUE THREAD

**Issue by EnkeyMC** [contember/engine#498]
Hello,

I have updated our project to version 1.3.3, but now I cannot transfer or import data from our production env (also 1.3) to my local environment (docker). In the logs I can see this:

```
                                   at /src/server.js:9408:23
                                   at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
                                   at async Mutex.execute (/src/server.js:2783:18)
                                   at async AcquiredConnection.query (/src/server.js:9380:16)
                                   at async /src/server.js:9369:18
                                   at async Mutex.execute (/src/server.js:2783:18)
                                   at async AcquiredConnection.scope (/src/server.js:9367:16)
                                   at async Transaction.query (/src/server.js:9249:16)
                                   at async connection.scope.eventManager.eventManager (/src/server.js:9237:18)
                                   at async /src/server.js:9369:18 {
                                 previous: error: duplicate key value violates unique constraint "subscribe_submission_form_id_email_key"
                                     at Parser.parseErrorMessage (/src/server.js:5796:99)
                                     at Parser.handlePacket (/src/server.js:5637:25)
                                     at Parser.parse (/src/server.js:5561:35)
                                     at Socket.<anonymous> (/src/server.js:5837:45)
                                     at Socket.emit (node:events:514:28)
                                     at Socket.emit (node:domain:489:12)
                                     at addChunk (node:internal/streams/readable:324:12)
                                     at readableAddChunk (node:internal/streams/readable:297:9)
                                     at Readable.push (node:internal/streams/readable:234:10)
                                     at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
                                   length: 331,
                                   severity: 'ERROR',
                                   code: '23505',
                                   detail: 'Key (form_id, email)=(3ac14470-69d0-4fbd-afb0-6f29d230642f, some.email@email.cz) already exists.',
                                   hint: undefined,
                                   position: undefined,
                                   internalPosition: undefined,
                                   internalQuery: undefined,
                                   where: undefined,
                                   schema: 'stage_live',
                                   table: 'subscribe_submission',
                                   column: undefined,
                                   dataType: undefined,
                                   constraint: 'subscribe_submission_form_id_email_key',
                                   file: 'nbtinsert.c',
                                   line: '663',
                                   routine: '_bt_check_unique'
                                 },
                                 code: '23505',
                                 originalMessage: 'duplicate key value violates unique constraint "subscribe_submission_form_id_email_key"',
                                 sql: 'INSERT INTO "stage_live"."subscribe_submission" ("id", "created_at", "email", "form_id", "locale_id", "first_name", "last_name", "datacruit_id", "consent", "url", "attachment_file_name", "message") VALUES\n' +
                                   <data truncated by me to keep it consise and to not leak sensitive data>
                                   ... 1304 more items
                                 ],
                                 constraint: 'subscribe_submission_form_id_email_key',
                                 table: 'subscribe_submission'
                               }
```

From this I understand that it's trying to import data that contain duplicate form_id and email pair, which is not allowed by `subscribe_submission_form_id_email_key` constraint. However, I have no idea where this constraint came from as I cannot find it in the migrations nor the code itself and if I import the data with contember-engine@1.2.2 it goes through just fine.

**What works:**

1. Start fresh docker with contember-engine@1.2.2
2. Run migrations
3. Transfer data from production
4. Data successfully transferred
5. Kill the docker containers, but keep volumes
6. Start docker with contember-engine@1.3.3
7. Transfer data from production
8. Data successfully transferred

**What does _NOT_ work:**

1. Start fresh docker with contember-engine@1.3.3
2. Run migrations
3. Transfer data from production
4. Server error (the one I posted above)

Not sure why the data can be imported when there is already some data (I believe I tried just upgrading from 1.2 to 1.3 without any data import beforehand and it also failed to import data), but when you have fresh 1.3 it fails.

If you need any more info from me, let me know.

**Comment by matej21** [comment]
Hello, thanks for reporting the issue. 
There was an issue with migrations not removing the unique constraint in some cases. I've created PR which addresses this issue #499. I expect I release the fix tomorrow.

**Comment by matej21** [comment]
Hi, sorry for the delay. The issue is fixed in 1.3.4 release, which is also deployed to the Contember Cloud. Please let me know, if it works correctly.

**Comment by EnkeyMC** [comment]
It works now, thanks for the fix.
